doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -400,6 +400,8 @@ Networking samples
 
   * Added ``DNS_SD_REGISTER_TCP_SERVICE`` so that mDNS services can locate and address the server using its hostname.
 
+  * Added support for native_sim board.
```

Comment:
```suggestion
  * Added:  

    * ``DNS_SD_REGISTER_TCP_SERVICE`` so that mDNS services can locate and address the server using its hostname.
    * Support for the :ref:`native simulator <zephyr:native_sim>` board.
```